packages/core/src/index.js

```diff
@@ -35,7 +36,10 @@ export const createParser = config => {
       if (!config[key]) continue
       const sx = config[key]
       const raw = props[key]
-      const scale = get(props.theme, sx.scale, sx.defaults)
+      const scale =
+        typeof sx.scale === 'object'
+          ? sx.scale
+          : get(props.theme, sx.scale, sx.defaults)
```

Comment:
Thanks for working on this! I *think* this doing the same thing as `sx.defaults` does, which picks up from the `defaultScale` option, right? I might be missing something about this change though